packages/waku/src/router/client.ts

```diff
@@ -102,19 +102,36 @@ type ChangeRoute = (
   },
 ) => Promise<void>;
 
+type ChangeRouteEvent = 'start' | 'complete';
+
+type ChangeRouteCallback = (route: RouteProps) => void;
+
+const EMPTY_ROUTE_CHANGE_LISTENER: Record<
+  'on' | 'off',
+  (event: ChangeRouteEvent, handler: ChangeRouteCallback) => void
+> = {
+  on: () => {},
+  off: () => {},
+};
```

Comment:
Something like this?
```suggestion
const DUMMY_ROUTE_CHANGE_LISTENER: Record<
  'on' | 'off',
  (event: ChangeRouteEvent, handler: ChangeRouteCallback) => void
> = {
  on: () => notAvailableInServer('routeChange:on'),
  off: () => notAvailableInServer('routeChange:off'),
};
```

Can we define this close to `INTERNAL_ServerRouter`?
`DUMMY_` sounds misleading. Suggestions?

Reply:
mock?

Reply:
> mock?

maybe.

Can we define this close to INTERNAL_ServerRouter?

Reply:
> mock?

maybe.

Can we define this close to INTERNAL_ServerRouter?